lib/wallet.py

```diff
@@ -695,16 +695,17 @@ def get_addr_received(self, address):
     def get_addr_balance(self, address):
         received, sent = self.get_addr_io(address)
         c = u = x = 0
+        minconf = config.get('minconf', 1)
         local_height = self.get_local_height()
         for txo, (tx_height, v, is_cb) in received.items():
             if is_cb and tx_height + COINBASE_MATURITY > local_height:
                 x += v
-            elif tx_height > 0:
```

Comment:
tx_height is a block height where the tx was included. 0 and -1 have special meanings signalling the tx is unconfirmed. your change won't do what you want.